src/context/WalletProvider/WalletViewsSwitch.tsx

```diff
@@ -72,6 +72,27 @@ export const WalletViewsSwitch = () => {
     }
   }, [history, initialRoute])
 
+  /**
+   * Memoize the routes list to avoid unnecessary re-renders unless the wallet changes
+   */
+  const walletRoutesList = useMemo(
+    () =>
+      type
+        ? SUPPORTED_WALLETS[type].routes.map(route => {
+            const Component = route.component
+            return !Component ? null : (
+              <Route
+                exact
+                key={`${type}_${String(route.path)}`}
```

Comment:
@gomesalexandre to check we don't have flicker regression

Reply:
Confirmed this one doesn't produce a regression after reverting `src/Routes/Routes` but should still use a static key for the same reason.

Additionally - this is copied from the current implementation so not expected to be changed here but keeping this for reference: with the same idea of keeping things nicely mounted, we will want to remove the `!Component ? null : <Route {...props} />` and  add that logic either in the `render` prop here, or as children instead (both will work similarly):

https://github.com/shapeshift/web/blob/9a3e57daba131347450ef9432afa67cd0766c1a3/src/context/WalletProvider/WalletViewsSwitch.tsx#L88

Being conditional with `<Route />` elements being rendered in the context of a JSX iteration is a pattern we don't want to spread as it could cause some bugs if done in a different context, the declarative logic of react-router expects `<Route />` components to be stable and re-render.

This particular case is functionally fine given `SUPPORTED_WALLETS[type].routes` is static and we will never be in a situation where `route.component` is initially undefined, then defined or the other way around.

Reply:
@cjthompson Let's bring back the static key here